ui/src/layouts/BuildPublication.tsx

```diff
@@ -291,7 +291,9 @@ const BuildPublication: React.FC<BuildPublicationProps> = (props) => {
         [checkRequiredApproval, isReadyToPreview, store]
     );
 
-    const hasCoAuthors = store.coAuthors.length > 1 ? true : false;
+    const hasCoAuthors = !store.coAuthors
```

Comment:
Please pay attention to naming. The name needs to match the checking, otherwise it can be very confusing and can lead to bugs if other devs will use this boolean just by it's name. You're checking if not all the co-authors have been invited, therefore the name: 'hasUninvitedCoAuthors'.